homeassistant/components/deconz/deconz_event.py

```diff
@@ -26,9 +23,8 @@ def async_add_sensor(sensors):
             if not gateway.option_allow_clip_sensor and sensor.type.startswith("CLIP"):
                 continue
 
-            if (
-                sensor.type not in Switch.ZHATYPE
-                or sensor.uniqueid in gateway.entities[EVENT]
+            if sensor.type not in Switch.ZHATYPE or sensor.uniqueid in (
```

Comment:
It's now a generator expression. I'm not sure that supports membership test. Use curly brackets to make it a set.

Reply:
>.< thanks, its getting late